### DISCUSSION TITLE
Automation triggered by Postgres row update

### DISCUSSION THREAD

**Discussion by JCV67** [Budibase/budibase#14049] [Help]
Hello everyone,

Automations triggered by Postgres row updates stopped working.
Tried to substitute it by app action triggered automations but don't work either.
At the same time a Postgres query executed after row saving started causing an error message.

Self hosted, the errors started a few updates ago but reversing versions did not solved the problem.

Could this be caused by error accumulation on an old BB installation?

Would appreciate your help 
Thank you

**Comment by JCV67** [comment]
Found out that this is happening only in one table that has relationship columns to 3 different tables. I'm nor being able of rebuilding the relationships using existent relationship data.

**Reply by ConorWebb96** [reply]
Could this be related to: https://github.com/Budibase/budibase/issues/13820?
